src/main/java/org/graylog2/plugins/slack/callback/SlackAlarmCallback.java

```diff
@@ -52,6 +53,17 @@ public void call(Stream stream, AlertCondition.CheckResult result) throws AlarmC
             message.addAttachment(new SlackMessage.AttachmentField("Stream Description", stream.getDescription(), false));
         }
 
+        // Add custom fields
+        if(!configuration.getString("ck_show_fields").isEmpty()){
+
+            MessageSummary msgSum = result.getMatchingMessages().get(0);
```

Comment:
This would only add the information from the first message as attachment to the Slack message. An alert can contain multiple messages.